server/src/main/java/org/opensearch/common/Stream.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.common;
+
+import java.io.InputStream;
+
+/**
+ * Model composed of an input stream, the total content length and offset
+ */
+public class Stream {
```

Comment:
This class is just temporarily present till repository plugin PRs are merged. We have changed its name in those PRs to covey proper meaning. This is not a stream class and instead it is just a wrapper over the raw stream which also contains its length and starting offset. We wrap the raw stream present in Stream class with encrypting stream. During this process, offset and length attributes of the supplied stream are required. 
I need the stream calls here temporarily so that we don't get any error during compile phase in build.

Reply:
This is used [here](https://github.com/opensearch-project/crypto/pull/2/files#diff-f11120a462e806fd84ae933f23a6ad7ba766096630a8cc0edb618335b86b65beR143)